mail_activity_board/__manifest__.py

```diff
@@ -8,7 +8,8 @@
     "development_status": "Beta",
     "category": "Social Network",
     "website": "https://github.com/OCA/social",
-    "author": "SDi, David Juaneda, Sodexis, ACSONE SA/NV, Odoo Community Association (OCA)",
+    "author": """SDi, David Juaneda, Sodexis, ACSONE SA/NV,
+        Odoo Community Association (OCA)""",
```

Comment:
Don't make literal multi-line strings, or the spaces and newlines will be part of the author field:

```suggestion
    "author": "SDi, David Juaneda, Sodexis, ACSONE SA/NV,"
    "Odoo Community Association (OCA)",
```